### PR TITLE
Added tutorial for RadioButton and CheckBox UI 

### PR DESCRIPTION
**Pull request by PaulNicolasHunter**
This is the first part of the tutorial in reference to issue number #205.
This tutorial demonstrates the use of RadioButton UI, we can toggle the colors of the cube using the given options.

### CODE REVIEW

File: docs/tutorials/01_ui/viz_radio_buttons.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+###############################################################################
+# Show Manager
+# ============================================================================
+#
+# Now that all the elements have been initialised, we add them to the show
+# manager.
+
+current_size = (800, 800)
+show_manager = window.ShowManager(size=current_size, title="DIPY Cube Example")
```
[the remark this answers is not in the document]
Comment (PaulNicolasHunter) [reply docs/tutorials/01_ui/viz_radio_buttons.py]: Got it, I'll push the suggested changes along with part 2 of this tutorial.

**Comment by PaulNicolasHunter** [comment]
@skoudoro, I've created the tutorial as you suggested, kindly, let me know if you want me to change anything or improvise it further.
cheers.

File: docs/tutorials/01_ui/viz_check_boxes.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+
+cube = cube_maker(color=(0, 0, 1), size=(20, 20, 20), center=(15, 0, 0))
+sphere = sphere_maker(color=(0, 0, 1), radius=11.0, center=(50, 0, 0),
+                      theta_resolution=360, phi_resolution=360)
+cone = cone_maker(color=(0, 0, 1), radius=10.0, center=(-20, -0.5, 0),
+                  height=20)
+arrow = arrow_maker(color=(0, 0, 1), start_point=(0, 25, 0),
+                    end_point=(40, 25, 0), shaft_resolution=50,
+                    tip_resolution=50)
```
Comment (skoudoro) [inline docs/tutorials/01_ui/viz_check_boxes.py]: Can you remove `cube_maker`, `sphere_make`r, `cone_maker`, `arrow_make`r and replace them by 
`actor.cone`, `actor.cube`, `actor.sphere`, `actor.arrow`.

you need to import `actor` module on the top

File: docs/tutorials/01_ui/viz_check_boxes.py [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+        cone_actor.GetProperty().SetColor(color)
+    return cone_actor
+
+
+########################################################################
+# Add an arrow to the scene.
+# =========================
+
+
+def arrow_maker(color=(1, 1, 1), start_point=(0, 25, 0),
```
Comment (skoudoro) [inline docs/tutorials/01_ui/viz_check_boxes.py]: no need of this function, it is on the `fury.actor` module. Please, remove it.

File: docs/tutorials/01_ui/viz_check_boxes.py [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+
+    return sphere_actor
+
+
+########################################################################
+# Add a cone to the scene.
+# =======================
+
+
+def cone_maker(color=(1, 1, 1), radius=5.0, center=(0, 0, 0),
```
Comment (skoudoro) [inline docs/tutorials/01_ui/viz_check_boxes.py]: no need of this function, it is on the `fury.actor` module. Please, remove it.

File: docs/tutorials/01_ui/viz_check_boxes.py [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+
+    return cube_actor
+
+
+########################################################################
+# Add a sphere to the scene.
+# =========================
+
+
+def sphere_maker(color=(1, 1, 1), radius=5.0, center=(0, 0, 0),
```
Comment (skoudoro) [inline docs/tutorials/01_ui/viz_check_boxes.py]: no need of this function, it is on the `fury.actor` module. Please, remove it.

File: docs/tutorials/01_ui/viz_check_boxes.py [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+"""
+
+from fury import ui, window
+
+########################################################################
+# Add a cube to the scene.
+# =======================
+
+
+def cube_maker(color=(1, 1, 1), size=(0.2, 0.2, 0.2), center=(0, 0, 0)):
```
Comment (skoudoro) [inline docs/tutorials/01_ui/viz_check_boxes.py]: no need of this function, it is on the `fury.actor` module. Please, remove it.

File: docs/tutorials/01_ui/viz_check_boxes.py [hunk 7]
```diff
@@ -0,0 +1,251 @@
...
+Figure and Color Control using Check boxes and Radio Buttons
+============================================================
+
+This example shows how to use the UI API. We will demonstrate how to
+create a cube and control its color using radio buttons.
+
+First, some imports.
+"""
+
+from fury import ui, window
```
Comment (skoudoro) [inline docs/tutorials/01_ui/viz_check_boxes.py]: Please, import `actor` module too

File: docs/tutorials/01_ui/viz_check_boxes.py (hunk 2 above)
Comment (PaulNicolasHunter) [reply docs/tutorials/01_ui/viz_check_boxes.py]: That's cool, I'll have a look at it.
But then why are we not using `actor.cube` in other tutorials as well?

Comment (skoudoro) [reply docs/tutorials/01_ui/viz_check_boxes.py]: Just because they are old and they need to be updated :smile:

Comment (PaulNicolasHunter) [reply docs/tutorials/01_ui/viz_check_boxes.py]: cool, I was looking at the `actor` earlier while coding the tutorials and was wondering why are we not incorporating this instead.
In this case, I should it's the `viz_ui.py` and `viz_ui_slider.py` as well, maybe in a separate PR.

Comment (PaulNicolasHunter) [reply docs/tutorials/01_ui/viz_check_boxes.py]: @skoudoro I'm unable to toggle colors of the figures when using `actor.cube`.
I've checked the value is being set(using cube.GetProperty().GetColor()) but it's not reflecting on the window.
what could be the reason? maybe the actors which are returned are the reason?

Comment (skoudoro) [reply docs/tutorials/01_ui/viz_check_boxes.py]: Indeed, it is much more complicated to update the colors. We need to create some functions/interfaces to make it easier. So, keep it as it is for the moment and just do the other corrections.
To get the color, you can use `cube.GetMapper().GetColorMapColors()` but it is not convenient to update it. It needs some thinking.
Thank you @PaulNicolasHunter !

Comment (PaulNicolasHunter) [reply docs/tutorials/01_ui/viz_check_boxes.py]: Alright, I understand @skoudoro, I'll try to dig more into it and find a workaround as soon as possible.

**Pull request merged**